app/src/main/res/values/dimens.xml

```diff
@@ -42,6 +42,7 @@
     <dimen name="sp_8">8sp</dimen>
     <dimen name="dp_25">25dp</dimen>
     <dimen name="dp_180">180dp</dimen>
+    <dimen name="sp_25">25sp</dimen>
```

Comment:
WARNING: [UnusedResources] The resource `R.dimen.sp_25` appears to be unused
 Unused resources make applications larger and slow down builds.